### ISSUE TITLE
Difficulty accessing links in Slideshow mode in PowerPoint

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#7288]
In PowerPoint (2016 at least), when running a slideshow which has clickable links, the links can be accessed by using TAB to move through links on a slide and then ENTER to activate a link.

NVDA does not read anything when TAB is pressed, which can make it difficult to know that a link has focus, or which link if there are more than one.  Ideally, when a link (on its own line) is read with the down arrow key, it should then be possible to activate it with ENTER, as can be done in other places (e.g. in web browsers).

To reproduce:
1. Open PowerPoint.
2. Add a URL to a slide (press ENTER after typing the URL to convert to an activatable link).
3. Press F5 to run the slideshow.

If you press ENTER at any point (eg when the URL is reported), the slideshow advances to the next slide (or the end of the slideshow).

To activate the link, press TAB to move to it, then ENTER.  The link opens in the browser, but NVDA does not report anything when TAB is pressed.

**Comment by Adriani90** [comment]
This is still reproducible in NVDA 2019.1.1.

**Comment by Qchristensen** [comment]
Still present in NVDA 2019.2.1 and alpha-19346,0c0d88aa 

No difference was found by editing the link (the same as editing links in Word) and changing the text to display or screen tip.

Pressing TAB, although it appears to move the focus (the link has a faint dotted outline), does not change NVDA's focus, navigator object or review cursor location.

The link does read as part of the text (eg with arrows), but is not read when using TAB.

**Comment by Sylvie42** [comment]
The problem is still there with NVDA 2022.4 and Microsoft Office 365, Powerpoint version: MSO (version 2301 build 16.0.16026.200002 64 bits).

In SlideShow, Links are not announced and it is not possible to access the links or enable them.

**Comment by Adriani90** [comment]
@michaelDCurran is it actually possible for NVDA to build a virtual document for slide show in presentation software like Powerpoint? This could be a topic for discussions with Microsoft, if they can provide a virtual buffer for powerpoint as it is possible for Microsoft Word and Excel. This could apply only for slide show mode in Powerpoint, and NVDA could access the slide elements with quick navigation keys like in other Microsoft office Products. This would solve a lot of problems such as accessing tables, headings etc. in slide show mode.

**Comment by RuturajL** [comment]
I have verified this issue with NVDA version 2024.1 and Office 365 Powerpoint desktop.
Links are not recognized by NVDA and it's not possible to activate them in presentation mode.
In edit mode, it is possible to read and open them from the applications menu.
This is an important issue as PPT is used by students and professionals around the world to share and understand information.
There's no workaround to my knowledge, in situations where edit access to file is not available.
